### PR TITLE
Issue #781: Improves usability of Field generation

### PR DESCRIPTION
**Pull request by heddn**

### CODE REVIEW

File: templates/module/src/Plugin/Field/FieldFormatter/fieldformatter.php.twig [hunk 1]
```diff
@@ -43,6 +43,7 @@ class {{ class_name }} extends FormatterBase {% endblock %}
    * {@inheritdoc}
    */
   public function settingsForm(array $form, FormStateInterface $form_state) {
+    $elements = array();
```
Comment (jmolivas) [inline templates/module/src/Plugin/Field/FieldFormatter/fieldformatter.php.twig]: May we use show syntax array 
```
$elements = [];
```

**Comment by heddn** [comment]
This changes things around a little to make the boilerplate that is generated hit the 80% target. Most of the time I'm creating a field, I want a text field. If this isn't the target desired, I can scale things back and leave only the shell.

**Comment by heddn** [comment]
The last commit fixes some issues with the annoations. It was never printing the field_types in the annotations for a field generator. Some of the strings were copy/pasted for formatter using the language from a widget. And #780 doesn't seem to have merged for some reason. So, re-adding that functionality.

**Comment by heddn** [comment]
OK, this PR is ready for merge now. I'm going to work on something else next.

**Comment by jmolivas** [comment]
@heddn: Thanks I will give it a review and test and the merge. ping you if any questions arise.

**Comment by heddn** [comment]
As a bonus to testing this, I uncovered https://www.drupal.org/node/2552799.

**Comment by jmolivas** [comment]
@heddn: Thanks for the PR

**Pull request merged**